native/common/jp_gc.cpp

```diff
@@ -0,0 +1,182 @@
+#include <Python.h>
+#include "jpype.h"
+#include "pyjp.h"
+#include "jp_reference_queue.h"
+
+#ifdef WIN32
+#define USE_PROCESS_INFO
+#include <Windows.h>
+#include <psapi.h>
+#elif __APPLE__
+#define USE_TASK_INFO
+#include <unistd.h>
+#include <sys/resource.h>
+#include <mach/mach.h>
+#else
+// Linux doesn't have an available rss tally so use mallinfo
+#define USE_MALLINFO
+#include <malloc.h>
+#endif
+#define DELTA_LIMIT 10*1024*1024l
+#define HARD_LIMIT 200*1024*1024l
+
+namespace
```

Comment:
whats the matter with this empty namespace?

Reply:
Variables were there and it appears that I forgot to come back and remove it.